README.md

```diff
@@ -1,15 +1,245 @@
+
+<p align="center"> 
+<img src="./images/Somatic_variant_calling_SIH_modified.png" width="60%">
+</p> 
+
 
 ## Workflow description 
 
 ## User guide 
 
-## Component tools 
+To run this pipeline, you will need to prepare your input files, reference data, and clone this repository. Before proceeding, ensure Nextflow is installed on the system you're working on. To install Nextflow, see these [instructions](https://www.nextflow.io/docs/latest/getstarted.html#installation). 
+
+### 1. Prepare inputs
+
+To run this pipeline you will need the following inputs: 
+
+* Paired Tumor-Normal paired BAM files
+* Corresponding BAM index files  
+* Input sample sheet 
+
+This pipeline processes paired BAM files and is capable of processing multiple samples in parallel. BAM files are expected to be coordinate sorted and indexed (see [Fastq-to-BAM](https://github.com/Sydney-Informatics-Hub/Fastq-to-BAM) for an example of a best practice workflow that can generate these files).  
+
+You will need to create a sample sheet with information about the samples you are processing, before running the pipeline. This file must be **tab-separated** and contain a header and one row per sample. Columns should correspond to sampleID, BAM-N file-path, BAM-T file-path: 
+
+|sampleID|bam-N                   |bam-T                       |
+|--------|----------------------|--------------------------|
+|SAMPLE1 |/data/Bams/sample1-N.bam|/data/Bams/sample1-T.bam|
+|SAMPLE2 |/data/Bams/sample2-N.bam|/data/Bams/sample2-T.bam|
+
+When you run the pipeline, you will use the mandatory `--input` parameter to specify the location and name of the input file: 
+
+```
+--input /path/to/samples.tsv
+```
+
+### 2. Prepare the reference materials 
+
+To run this pipeline you will need the following reference files:
+
+* Indexed reference genome in FASTA format 
+
+You will need to download and index a copy of the reference genome you would like to use. Reference FASTA files must be accompanied by a .fai index file. If you are working with a species that has a public reference genome, you can download FASTA files from the [Ensembl](https://asia.ensembl.org/info/data/ftp/index.html), [UCSC](https://genome.ucsc.edu/goldenPath/help/ftp.html), or [NCBI](https://www.ncbi.nlm.nih.gov/genome/doc/ftpfaq/) ftp sites. You can use our [IndexReferenceFasta-nf pipeline](https://github.com/Sydney-Informatics-Hub/IndexReferenceFasta-nf) to generate indexes. 
+
+When you run the pipeline, you will use the mandatory `--ref` parameter to specify the location and name of the reference.fasta file: 
+
+```
+--ref /path/to/reference.fasta
+```
+
+
+### 3. Clone this repository 
+
+Download the code contained in this repository with: 
+
+```
+git clone https://github.com/Sydney-Informatics-Hub/Somatic-shortV-nf
+```
+
+This will create a directory with the following structure: 
+```
+Somatic-shortV-nf/
+├── LICENSE
+├── README.md
+├── config/
+├── images/
+├── main.nf
+├── modules/
+├── PBS_gadi_runPipeline.sh
```

Comment:
this is missing the `scripts/` directory. Should also move PBS execution script to this directory when ready.

Reply:
Added scripts/

Reply:
I will move the PBS execution script to the script directory after testing.